<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="bag_path" default="/seerep/seerep-data/rosbag-FR/20230524/wdh4_174812_20230524_aql13-tf-reprocessed_2023-05-25-10-22-53.bag"/>
    <arg name="classes_mapping_path" default="/seerep/seerep-data/rosbag-FR/20230524/label_seerep_fr.json"/>
    <arg name="hdf5FolderPath" default="/seerep/seerep-data/rosbag-FR/20230524"/>
    <arg name="project_frame_id" default="map"/>
    <arg name="project_name" default="testproject"/>
    <arg name="topic_image" default="/cama/classifier_gpu_node/cw_debug_images"/>
    <arg name="topic_camera_intrinsics" default="/cama/classifier_gpu_node/camera_info"/>
    <arg name="topic_detection" default="/cama/classifier_gpu_node/cw_detection_array"/>
    <arg name="detection_category" default="theCategory"/>
    <arg name="topic_tf" default="/tf"/>
    <arg name="topic_tf_static" default="/tf_static"/>
    <arg name="topic_geo_anchor" default="/anchor"/>
    <arg name="distance_camera_ground" default="0.5"/>
    <arg name="store_images" default="false"/>

    <node name="seerep_ros_communication_rosbagDumper" pkg="seerep_ros_communication" type="seerep_ros_communication_rosbagDumper" output="screen">
        <rosparam param="bag_path" subst_value="True">$(arg bag_path)</rosparam>
        <rosparam param="classes_mapping_path" subst_value="True">$(arg classes_mapping_path)</rosparam>
        <rosparam param="hdf5FolderPath" subst_value="True">$(arg hdf5FolderPath)</rosparam>
        <rosparam param="project_frame_id" subst_value="True">$(arg project_frame_id)</rosparam>
        <rosparam param="project_name" subst_value="True">$(arg project_name)</rosparam>
        <rosparam param="topic_image" subst_value="True">$(arg topic_image)</rosparam>
        <rosparam param="topic_camera_intrinsics" subst_value="True">$(arg topic_camera_intrinsics)</rosparam>
        <rosparam param="topic_detection" subst_value="True">$(arg topic_detection)</rosparam>
        <rosparam param="detection_category" subst_value="True">$(arg detection_category)</rosparam>
        <rosparam param="topic_tf" subst_value="True">$(arg topic_tf)</rosparam>
        <rosparam param="topic_tf_static" subst_value="True">$(arg topic_tf_static)</rosparam>
        <rosparam param="topic_geo_anchor" subst_value="True">$(arg topic_geo_anchor)</rosparam>
        <rosparam param="distance_camera_ground" subst_value="True">$(arg distance_camera_ground)</rosparam>
        <rosparam param="store_images" subst_value="True">$(arg store_images)</rosparam>
    </node>
</launch>
